src/components/FeatureFlagRenderer.js

```diff
@@ -16,15 +16,39 @@ export default class FeatureFlagRenderer extends Component {
   constructor (props:Props) {
     super(props);
 
+    const {
+      flagKey,
+      ldClientConfig: {
+        clientOptions: {
+          bootstrap
+        }
+      }
+    } = this.props;
+
     this.state = {
       checkFeatureFlagComplete: false,
-      flagValue: false
+      flagValue: bootstrap && bootstrap.hasOwnProperty(flagKey) ? bootstrap[flagKey] : false
     };
+
+    // this.checkFeatureFlag = this.checkFeatureFlag.bind(this);
+    // this.listenFlagChangeEvent = this.listenFlagChangeEvent.bind(this);
+    this.setStateFlagValue = this.setStateFlagValue.bind(this);
```

Comment:
This is mostly unrelated to this PR, but I can't find anything else to complain about.

The weird issue that we've been seeing with hot-loaded code and undefined variables in events was due to constructor binding.  Also using the constructor scope in the function, which this code doesn't do.  But regardless, these can be replaced with the auto-binding arrow functions.  It's the current preferred style anyway, I am told.

```
setSstateFlagValue = (flagValue) => {
...
```
And the constructor binding can be deleted.